### ISSUE TITLE
about the mask for computing loss

### ISSUE THREAD

**Issue by shuluoshu** [ialhashim/DenseDepth#84]
Hi @ialhashim 
 I 've got a dataset that only have valid depth on some regions. I only want to compute loss on pixels that have valid depth and ignore other pixels when computing loss, I am not familiar with Keras, I want to know how to pass a Mask into the loss, could your show me the example for such case, Thank you so much!

**Comment by ibraheem-nt** [comment]
Look at examples in Tensorflow, these usually can be easily adapted to Keras models. Many other depth estimation works do that, e.g. (https://github.com/cogaplex-bts/bts/blob/master/bts.py#L366).

**Comment by ialhashim** [comment]
Look at examples in Tensorflow, these usually can be easily adapted to Keras models. Many other depth estimation works do that, e.g. (https://github.com/cogaplex-bts/bts/blob/master/bts.py#L366).
